src/app/shared/components/template/components/audio/audio.component.ts

```diff
@@ -30,19 +31,18 @@ interface IAudioParams {
    * Will be mirrored to be used as the reqind icon. Default icon is ion's "play-forward"
    * */
   forwardIconAsset: string;
-  /** TEMPLATE PARAMETER: "help_icon_asset". The path to an svg to override the default help icon. */
-  helpIconAsset: string;
-  /** TEMPLATE PARAMETER: "help_text". Text to be displayed as a tooltip when clicking the "help" icon.
-   * Icon and tooltip will not be displayed if value not provided. Default null */
-  helpText: string;
+  /** TEMPLATE PARAMETER: "show_info_button". Should show the info button. Default false (unless info_icon_asset is provided) */
+  showInfoButton: boolean;
```

Comment:
From an authoring perspective it would be quite nice to just provide the `transcript_text` as a string to the audio component, and the audio component would handle showing that text as a pop-up. 

This is because I would typically store the transcript text in a data list on the same row as the audio asset path, so it's easy to access when I'm also providing the asset path. It's a bit more tedious to set up a separate template generator to create all the transcript pop-up templates.

Reply:
I think probably just start by adding support for text and not a template. Then you can just use an inline modal with the content wrapped in an `if()` block on the audio player html to popup, and not bother with the template popup/nav service and bindings
https://ionicframework.com/docs/api/modal#inline-modals-recommended

Reply:
Thanks, that's helpful. Although it seems inline modals are only supported in Ionic v6+ (we're on 5). I would like to update the Ionic version anyway, but additionally after discussion with @esmeetewinkel we agreed that it would be unsatisfactory if the launched pop-up looked different from our existing pop-ups elsewhere, so it would make sense to integrate with our existing pop-up component. We agreed that it would make sense to merge this PR as-is for now, and address in a follow-up